### PR TITLE
Improved logging for failed traces.

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: lib/hooks/core/hook-http.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ function requestWrap(request) {
 
     if (!root) {
       agent.logger.warn(
-        'Cannot trace outbound http requests outside of a supported http framework.');
+        'Cannot trace outbound http requests outside of a supported http framework. ' +
+        (typeof options === 'string' ? options : JSON.stringify(options)));
```
Comment (ofrobots) [inline lib/hooks/core/hook-http.js]: Extra information should be logged at higher logging levels.

Comment (ofrobots) [inline lib/hooks/core/hook-http.js]: i.e. let's not do a stringify

Comment (matthewloring) [inline lib/hooks/core/hook-http.js]: Fixed.

**Comment by ofrobots** [comment]
lgtm.

**Pull request merged**